Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
       parser (>= 3.1.1.0)
-    rubocop-performance (1.13.3)
+    rubocop-performance (1.14.3)
       rubocop (>= 1.7.0, < 2.0)
       rubocop-ast (>= 0.4.0)
     ruby-progressbar (1.11.0)
     rubyzip (2.3.2)
-    selenium-webdriver (4.1.0)
+    selenium-webdriver (4.4.0)
       childprocess (>= 0.5, < 5.0)
       rexml (~> 3.2, >= 3.2.5)
-      rubyzip (>= 1.2.2)
-    sidekiq (6.4.1)
-      connection_pool (>= 2.2.2)
+      rubyzip (>= 1.2.2, < 3.0)
+      websocket (~> 1.0)
+    sidekiq (6.5.6)
+      connection_pool (>= 2.2.5)
       rack (~> 2.0)
-      redis (>= 4.2.0)
+      redis (>= 4.5.0, < 5)
     simple_form (5.1.0)
       actionpack (>= 5.2)
       activemodel (>= 5.2)
-    sshkit (1.21.2)
+    sshkit (1.21.3)
       net-scp (>= 1.1.2)
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
-    standard (1.9.1)
-      rubocop (= 1.26.1)
-      rubocop-performance (= 1.13.3)
-    stimulus-rails (1.0.4)
+    standard (1.16.1)
+      rubocop (= 1.35.1)
+      rubocop-performance (= 1.14.3)
+    stimulus-rails (1.1.0)
       railties (>= 6.0.0)
-    strscan (3.0.1)
+    strscan (3.0.4)
     thor (1.2.1)
-    timeout (0.2.0)
-    turbo-rails (1.0.1)
+    timeout (0.3.0)
+    turbo-rails (1.1.1)
       actionpack (>= 6.0.0)
+      activejob (>= 6.0.0)
       railties (>= 6.0.0)
-    tzinfo (2.0.4)
+    tzinfo (2.0.5)
       concurrent-ruby (~> 1.0)
-    unicode-display_width (2.1.0)
+    unicode-display_width (2.3.0)
```

Comment:
### [unicode-display_width](https://github.com/janlelis/unicode-display_width)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.1.0 → 2.3.0

[_[change-log](https://github.com/janlelis/unicode-display_width/blob/master/CHANGELOG.md), [source-code](https://github.com/janlelis/unicode-display_width), [gem-diff](https://my.diffend.io/gems/unicode-display_width/2.1.0/2.3.0)_]





<details>
<summary>Commits</summary>

A change of **13** commits. See the full changes on [the compare page](https://github.com/janlelis/unicode-display_width/compare/v2.1.0...v2.3.0).

These are the first 10 commits:
- (df5a634) [README: Update list of supported Rubies](https://github.com/janlelis/unicode-display_width/commit/df5a63473e8b8f46dcda4756b8abfdece7305810)
- (f7f564d) [GitHub CI: Update Rubies](https://github.com/janlelis/unicode-display_width/commit/f7f564d338e66890401ea9399a5e6f3276670980)
- (d5ba149) [Gemspec: Add MFA requirement](https://github.com/janlelis/unicode-display_width/commit/d5ba149c5105a3df5bfd031b3d0e454968aca8c9)
- (53a5c13) [Update license year](https://github.com/janlelis/unicode-display_width/commit/53a5c132c388d0dd8c7475bf8661838e1aac01fe)
- (cdbb5de) [Add *Hangul Jamo Extended-B* block to zero-width chars](https://github.com/janlelis/unicode-display_width/commit/cdbb5deff5f8db0284f47c371e2e0e3f8e4872da)
- (58d78ea) [Update CI Rubies](https://github.com/janlelis/unicode-display_width/commit/58d78ea91c0f41d646155bac14af5199e4162b73)
- (f4c7b3f) [Release v2.2.0](https://github.com/janlelis/unicode-display_width/commit/f4c7b3f5ca5f1ceba05e23fd7dded6276906bcf6)
- (d1d2bb1) [Update CI Rubies](https://github.com/janlelis/unicode-display_width/commit/d1d2bb17b60a4043957e0c85da27b1e41b43d036)
- (2b672d3) [Unicode 15.0](https://github.com/janlelis/unicode-display_width/commit/2b672d3142c84d4d938c002af1f0a085038ebeaa)
- (da13ef3) [Update Unicode version number](https://github.com/janlelis/unicode-display_width/commit/da13ef319cabcde4a9b0fba6290524d7d996de7e)


</details>